blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Sourcegraph 3.8: Introducing campaign management with Automation and LSIF-based code intelligence"
+author: Christina Forney
+publishDate: 2019-09-01T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.8
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+We're excited to announce Sourcegraph 3.8. This release contains lots of behind the scenes work, both foundational and forward looking. Sourcegraph 3.8 introduces faster and more precise [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md)-based code intelligence, and campaign monitoring, the first step towards [code change Automation](https://about.sourcegraph.com/product/automation). In addition to these new concepts, 3.8 introduces search result tabs for better search result discoverability, and the first step towards better monitoring for admins with built-in monitoring tools deployed with every instance.
+
+<div style="padding-left: 2rem">
+
+[**🤖 Introducing Automation: first stages of campaign management**](#introducing-automation-first-stages-of-campaign-management)<br />
+TO DO???
+
+[**🧠 Beta release of fast and precise LSIF-based code intelligence**](#beta-release-of-fast-and-precise-lsif-based-code-intelligence)<br />
+Faster, more precise code intelligence for TypeScript, Go, Python, and C/C++
+
+[**🔎 Search result tabs for improved resuslt type discoverability**](#search-result-tabs-for-improved-resuslt-type-discoverability)<br />
+
+[**📊 Out of the box monitoring of your Sourcegraph instance**](#out-of-the-box-monitoring-of-your-sourcegraph-instance)<br />
+
+[**📉 Behind the scenes: Even more benchmarking, automated testing, and QA**](#behind-the-scenes-even-more-benchmarking-automated-testing-and-qa)<br />
+
+[**🔦 Quickly toggle the browser extension on/off**](#quickly-toggle-the-browser-extension-onoff)<br />
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
```

Comment:
```suggestion
Every detail that changed in this release.
```

Reply:
Historically in release notes these sentences haven't had periods. Happy to change that for all the sub-sentences in the list if you think it would be better!

Reply:
Updating.